environments/gcp/flyte-core/README.md

```diff
@@ -0,0 +1,95 @@
+# Flyte on GCP
+
+Prerequisites:
+
+- [terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli#install-terraform)
+
+Procedure:
+
+1. Create a project on GCP and get its PROJECT_ID:
+
+    ```bash
+    gcloud projects list
+    ```
+>NOTE: learn how to setup the gcloud CLI [here](https://cloud.google.com/sdk/docs/initializing#initialize_the)
+
+2. Acquire credentials to access the new project:
+
+```bash
+gcloud auth application-default login
+```
+
+3. Create a bucket in the project and region where you will deploy Flyte, leaving public access off. 
+
+4. Go to `locals.tf` and change the following variables to your environment specifics:
+
+| Key      | Value |Notes |
+| ----------- | ----------- |-----|
+| `application`      | Use your own/leave default      |    This is just a label  |
+| `environment`  | Use your own/leave default    |  This is just a label    |
+| `project_id` | your GPC project ID |
+`dns-domain` | A DNS domain you own, so SSL certificates can be generated|
+|`region` | The GCP region you'll use |
+
+5. Save your changes.
+6. Go to `terraform.tf` and replace the name of the GCS bucket you created in step 2 in the appropiate section:
```

Comment:
Did you test whether this can be changed in `locals.tf` as well?

Reply:
Yep, I tried but it errors out, it doesn't allow variables there